packages/react-sdk/src/hooks/useFlowSchedule.ts

```diff
@@ -0,0 +1,539 @@
+import {arg, t} from "@onflow/fcl"
+import {useCallback} from "react"
+import {useFlowClient} from "./useFlowClient"
+import {useFlowMutate} from "./useFlowMutate"
+import {useFlowChainId} from "./useFlowChainId"
+import {CONTRACT_ADDRESSES, CADENCE_UFIX64_PRECISION} from "../constants"
+import {parseUnits} from "viem/utils"
+
+export enum TransactionPriority {
+  Low = 0,
+  Medium = 1,
+  High = 2,
+}
+
+export enum TransactionStatus {
+  Pending = 0,
+  Processing = 1,
+  Completed = 2,
+  Failed = 3,
+  Cancelled = 4,
+}
+
+export interface TransactionInfo {
+  id: bigint
+  priority: TransactionPriority
+  executionEffort: bigint
+  status: TransactionStatus
+  fees: {
+    value: bigint
+    formatted: string
+  }
+  scheduledTimestamp: number
+  handlerTypeIdentifier: string
+  handlerAddress: string
+}
+
+export interface TransactionInfoWithHandler extends TransactionInfo {
+  handlerUUID: bigint
+  handlerResolvedViews: {[viewType: string]: any}
+}
+
+export interface UseFlowScheduleArgs {
+  flowClient?: ReturnType<typeof useFlowClient>
+}
+
+export interface UseFlowScheduleResult {
+  // Lists all transactions for an account
+  // Equivalent to: flow schedule list <account> [--include-handler-data]
+  list: (
+    account: string,
+    options?: {includeHandlerData?: boolean}
+  ) => Promise<TransactionInfo[] | TransactionInfoWithHandler[]>
+
+  // Gets a transaction by ID
+  // Equivalent to: flow schedule get <transaction-id> [--include-handler-data]
+  get: (
+    transactionId: bigint,
```

Comment:
I think we have mixed `txId` and `transactionId` across the React SDK. We should probably be consistent. I'm for `transactionId` for clarity, so this is fine, but curious what everyone else thinks. cc @jribbink

Reply:
Yep, just updated to string for consistency